tools-v2/pkg/config/config.go

```diff
@@ -167,6 +168,34 @@ func AlignFlagsValue(caller *cobra.Command, callee *cobra.Command, flagNames []s
 	}
 }
 
+type stringSlice struct {
+	value []string
+	change bool
```

Comment:
seem useless?

Reply:
> seem useless?

This is to make the flag names of the calling command and the called command consistent, but one is prepared when the string slice and the other is string. There is a problem with the design of pflag, which can be avoided by this method.